src/errors.rs

```diff
@@ -746,6 +752,23 @@ fn on_install_packages_error(error: InstallPackagesError) -> ErrorMessage {
     }
 }
 
+fn on_detect_error(error: DetectError) -> ErrorMessage {
+    match error {
+        DetectError::CheckExistsAptfile(file, e) | DetectError::CheckExistsProjectToml(file, e) => {
+            let file = file_value(file);
+            create_error()
+                .error_type(UserFacing(SuggestRetryBuild::No, SuggestSubmitIssue::No))
```

Comment:
Should this be `SuggestRetryBuild::Yes` instead? (this is a transient error that a retry may fix, eg if bad build host)